master/master.cfg

```diff
@@ -107,17 +107,19 @@ WorkerConfig = namedtuple('WorkerConfig', ['max_builds', 'j', 'arch', 'bits', 'o
 # is definitely not what we want.
 _NPROC_PLUS_2 = Transform(lambda x: f'{int(x)+2}', Interpolate("%(worker:numcpus)s"))
 
+# Using nproc+2 on the arm32 builds causes internal errors in gcc-armeabihf. Let's just use nproc.
+_NPROC = Interpolate("%(worker:numcpus)s")
```

Comment:
Can you reuse this in the definition of _NPROC_PLUS_2?

Reply:
PTAL